tools/downloader/src/open_model_zoo/model_tools/_common.py

```diff
@@ -110,3 +129,30 @@ def get_package_path(python_executable, package_name):
     # For a package, the file is __init__.py, so to get the package path,
     # take its parent directory.
     return file_path.parent
+
+def get_version():
+    if VERSION_FILE.is_file():
+        with VERSION_FILE.open('r') as version_file:
+            version = version_file.readline().rstrip('\n')
+        return version
```

Comment:
```suggestion
        return f'commit {version}'
```

This will make it easier to switch to a different (non-commit-based) versioning system in the future (if we want to), because we'll be able to identify the versioning system by the first word.

Reply:
Done